shared/versions.yml

```diff
@@ -70,18 +70,34 @@ couchdb:
 dataloader:
   upstream:
     repository: gpii/universal
-    tag: 20190522142238-4a52f56
+    tag: 20190610144430-9bd021f
   generated:
-    repository: gcr.io/gpii-common-prd/gpii/universal
-    sha: sha256:35eafa2a5fe1e5b4dd908596854fd46015a75c4f0758c743c2491868fe8b12fa
-    tag: 20190522142238-4a52f56
+    repository: gpii/universal
+    sha: sha256:42f238d64e3afec834a1da2b33b602be68e6acb437d623b2b5c559c61a9f6b2a
+    tag: 20190610144430-9bd021f
+flushtokens:
+  upstream:
+    repository: gpii/universal
+    tag: 20190610144430-9bd021f
+  generated:
+    repository: gpii/universal
+    sha: sha256:42f238d64e3afec834a1da2b33b602be68e6acb437d623b2b5c559c61a9f6b2a
+    tag: 20190610144430-9bd021f
+istio_proxy_manager:
```

Comment:
The TF module/Helm chart, as it's written at the moment wouldn't pick this up, as the image & it's version are hardcoded. I commented above that's it's not necessary use the istio image and I would drop this part in favor of using, for example, the `cloud_sdk` image.